falcon/request.py

```diff
@@ -893,19 +894,8 @@ def params(self):
     @property
     def cookies(self):
         if self._cookies is None:
-            # NOTE(tbug): We might want to look into parsing
-            # cookies ourselves. The SimpleCookie is doing a
-            # lot if stuff only required to SEND cookies.
             cookie_header = self.get_header('Cookie', default='')
-            parser = SimpleCookie()
```

Comment:
I guess you can remove import of `http_cookies`/`SimpleCookie`

Reply:
right, I did do that in an earlier commit but it seems to have broken the tox 'hug' env build, wanted to ask @kgriffs about that before removing the `SimpleCookie` import?